### PR TITLE
bug fix change VAT depends on country selection

### PR DESCRIPTION
**Pull request by DanielMaass**
-[CORE-802]

[CORE-802]: https://evannetwork.atlassian.net/browse/CORE-802

### CODE REVIEW

File: dapps/profile.vue/src/components/wallet/buy/BuyEve.ts [hunk 1]
```diff
@@ -107,6 +108,12 @@ export default class BuyEveComponent extends mixins(EvanComponent) {
     success: false,
   };
 
+  @Watch('contactForm.country.value')
+  onCountryChange(country) {
+    // change tax value depends on country select
+    this.taxValue = country === 'DE' && !this.contactForm.vat.value ? 19 : 0;
```
Comment (karladler) [inline dapps/profile.vue/src/components/wallet/buy/BuyEve.ts]: This looks like it's possible to buy from Germany without paying VAT. IMO this shouldn't be the behaviour.

Comment (karladler) [reply dapps/profile.vue/src/components/wallet/buy/BuyEve.ts]: see also: https://evannetwork.atlassian.net/browse/CORE-395

Comment (S3bb1) [inline dapps/profile.vue/src/components/wallet/buy/BuyEve.ts]:
```suggestion
    this.taxValue = country === 'DE' ? 19 : 0;
```

**Review by S3bb1 — APPROVED** [review]

**Comment by datrinh** [comment]
Could we at least put the hardcoded numbers into constant variables please? :)

**Pull request merged**